releaf.gemspec

```diff
@@ -38,5 +38,4 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'spring-commands-rspec'
   s.add_development_dependency 'roo'
 
-  s.required_ruby_version = '>= 2.2.0'
```

Comment:
I guess it's better to keep this in releaf.gemspec, as it is "mother" of all gemspec in releaf gem.